app/services/candidate_submission.rb

```diff
@@ -0,0 +1,71 @@
+class CandidateSubmission
+  include Candidate
```

Comment:
Should this be a concern? I don't see it used elsewhere and it's just a bunch of constants - I think the naming and placement is confusing as well, it's not a `Candidate` model

Reply:
yep, have moved other mods also